### PR TITLE
npm scripts: add `aggregate-output`

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

**Review by mdo — APPROVED** [review]

**Comment by XhmikosR** [comment]
I want to check this a little more; maybe we could skip adding the flag in some cases.

**Comment by XhmikosR** [comment]
This works nice but seems to break the CLI colors :/

**Comment by XhmikosR** [comment]
I can no longer reproduce the colors issue on Windows. I'll give in another go later to be sure.

File: package.json [hunk 1]
```diff
@@ -20,46 +20,46 @@
     "Twitter, Inc."
   ],
   "scripts": {
-    "start": "npm-run-all --parallel watch docs-serve",
+    "start": "npm-run-all --aggregate-output --parallel watch docs-serve",
```
Comment (alpadev) [inline package.json:23]: Mh this prevents the output of long living tasks like docs-serve 🤔

Comment (XhmikosR) [reply package.json:23]: Good catch! I'll revert this one.

**Comment by mdo** [comment]
@XhmikosR Feel free to merge this whenever you're up for it!

**Comment by XhmikosR** [comment]
I'm going to merge this and then backport it. If anyone notices any issues please CC me.

**Pull request merged**